src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q9/ReversePlate.java

```diff
@@ -0,0 +1,30 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q9;
+
+/*
+Traffic speed cameras are used to catch drivers who speed. One of the issues with this system is that it reads the plate
```

Comment:
Also, I don't imagine this will work with negative numbers, so we should probably specify that the numbers will be positive.